Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/AnalyticsConfigMapper.kt

```diff
@@ -4,20 +4,59 @@ import dagger.Reusable
 import de.rki.coronawarnapp.appconfig.AnalyticsConfig
 import de.rki.coronawarnapp.appconfig.SafetyNetRequirements
 import de.rki.coronawarnapp.appconfig.SafetyNetRequirementsContainer
+import de.rki.coronawarnapp.appconfig.internal.ApplicationConfigurationInvalidException
 import de.rki.coronawarnapp.server.protocols.internal.v2.AppConfigAndroid
 import javax.inject.Inject
 
 @Reusable
 class AnalyticsConfigMapper @Inject constructor() : AnalyticsConfig.Mapper {
     override fun map(rawConfig: AppConfigAndroid.ApplicationConfigurationAndroid): AnalyticsConfig {
-        // TODO
-        return AnalyticsConfigContainer()
+        if (!rawConfig.hasPrivacyPreservingAnalyticsParameters() &&
+            !rawConfig.privacyPreservingAnalyticsParameters.hasCommon() &&
+            !rawConfig.privacyPreservingAnalyticsParameters.hasPpac()
+        ) {
+            throw ApplicationConfigurationInvalidException(
```

Comment:
Is the expected behavior to throw here (and discard the whole config), or would it be okay to return a default of "analytics disabled", as analytics being disabled would still allow the rest of the app to function?

Reply:
i talked to @mlenkeit about this, he mentioned that the analytics config part of the app config should not be optional so we can assume a broken config if no analytics config is provided